ui/pages/notifications/index.scss

```diff
@@ -96,13 +96,19 @@
     }
 
     .unread {
+      flex-shrink: 0;
       background-color: var(--color-primary-default);
     }
 
     &__details {
       display: flex;
       flex-direction: column;
       margin-left: 12px;
+      width: 100%;
```

Comment:
```suggestion
```
I don't think this is needed anymore after using `overflow-wrap`?

Reply:
@FrederikBolding, I have attached screenshots below. Without setting the width to 100% in this case, overflow-wrap does not work correctly.

![image](https://user-images.githubusercontent.com/60202692/235631429-09e0ed0a-2080-4c3b-8876-d9a65a9f8d45.png)
![image](https://user-images.githubusercontent.com/60202692/235631513-c6cce8bd-2b22-4e64-b553-b6fd6233e8b1.png)